modules/spring-config/src/main/java/org/mule/runtime/config/api/dsl/model/metadata/ComponentBasedIdHelper.java

```diff
@@ -63,7 +64,9 @@ static String sourceElementNameFromSimpleValue(ComponentAst element) {
     // There seems to be something missing in the mock model from the unit tests and this fails.
     // return MuleAstUtils.parameterOfType(elementModel, MuleStereotypes.CONFIG)
     // .map(p -> p.getValue().reduce(identity(), v -> v.toString()));
-    return elementModel.getRawParameterValue(CONFIG_ATTRIBUTE_NAME);
+    return ofNullable(elementModel.getParameter(CONFIG_ATTRIBUTE_NAME) != null
```

Comment:
shouldn't this be `ofNullable(elementModel.getParameter(CONFIG_ATTRIBUTE_NAME)).map(ComponnetParameterAst::getRawValue).orElse(null)`?

Reply:
You are right, but without the "orElse" because this method returns an optional